activities/Paint.activity/js/activity.js

```diff
@@ -51,6 +51,24 @@ define(["sugar-web/activity/activity","tutorial","webL10n","sugar-web/env","acti
       webL10n.language.code = language;
     });
 
+     // Export as PNG image
+		document.getElementById("save-image-button").addEventListener('click', function() {
```

Comment:
Respect indentation. SPACE instead of TAB here.

Reply:
In my Visual studio code, it is showing proper indentation. This problem occurs only when I push code to GitHub, due to different tab sizes used by GitHub and Visual Studio.
Thanks for pointing, I will cross check it in the next commit.